chef/cookbooks/barclamp/spec/libraries/barclamp_library_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "spec_helper"
+
+require_relative "../../libraries/barclamp_library"
+
+describe BarclampLibrary::Barclamp::Inventory::Disk do
+
+  before(:each) do
+    @chef_run = ::ChefSpec::Runner.new
+    @node = @chef_run.node
+    @node.default[:block_device] = {
+      dm0: { removable: "0" },
+      xvd1: { removable: "0" },
+      xvd2: { removable: "1" }
+    }
+  end
+
+  it "unclaimed method returns the proper number of unclaimed devices" do
+    a = BarclampLibrary::Barclamp::Inventory::Disk
+    expect(a).to receive(:`).exactly(5).times.and_return(`exit 1`)
```

Comment:
The backtick method runs the given command in a subshell: http://ruby-doc.org/core-2.4.1/Kernel.html#method-i-60

So here it's setting up a mock that will return the same output as `exit 1`.  But that's just the empty string, so I'm not sure what the benefit of the `exit 1` is here.